php_pdo_ibm_int.h

```diff
@@ -33,6 +33,8 @@
 #define SQL_XML -370
 #endif
 
+typedef unsigned long ulong;
```

Comment:
I think we should rather make sure that `ulong`is not used as in the other drivers.